src/classes/BGE_BatchGiftEntry_UTIL.cls

```diff
@@ -306,7 +306,7 @@ public with sharing class BGE_BatchGiftEntry_UTIL {
             String errorMsg = Label.bgeFLSError + ' [' + truncateList(missingPermissions, 3) + ']';
             AuraHandledException ex = new AuraHandledException(errorMsg);
             ex.setMessage(errorMsg);
-            throw ex;
+            // throw ex;
```

Comment:
Added back the thrown exception. As we discussed, I removed the method call in `BGE_DataImportBatchEntry_CTRL` that was performing crud/fls checking for the BGE table so that the permission checking only occurs from the SGE form load.